BeforeBuild.cmd

```diff
@@ -0,0 +1,35 @@
+::Configuration for AppVeyor build platform
+
+@ECHO off
+SETLOCAL
+
+if NOT exist "GeneratedKey.snk" (
+sn -k GeneratedKey.snk
+)
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo COMMAND sn -k FAILED
```

Comment:
Avoid ALL-CAPS in messages. It sounds like you're shouting :-) Instead:

```
echo Command "sn -k" failed.
```

... here and throughout.
```